### PR TITLE
Add /sdk/internal to CODEOWNERS

### PR DESCRIPTION
**Pull request by jhendrixMSFT**
Remove jhendrixMSFT from all track 2 content.

- [ ] The purpose of this PR is explained in this or a referenced issue.
- [ ] The PR does not update generated files.
   - These files are managed by the codegen framework at [Azure/autorest.go][].
- [ ] Tests are included and/or updated for code changes.
- [ ] Updates to [CHANGELOG.md][] are included.
- [ ] MIT license headers are included in each file.

[Azure/autorest.go]: https://github.com/Azure/autorest.go
[CHANGELOG.md]: https://github.com/Azure/azure-sdk-for-go/blob/main/CHANGELOG.md

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -18,6 +18,9 @@
 # PRLabel: %Azure.Identity
 /sdk/azidentity/         @rickwinter @chlowell
 
+# PRLabel: %Internal
+/sdk/internal/           @rickwinter @jhendrixMSFT @richardpark-msft @seankane-msft
```
Comment (RickWinter) [inline .github/CODEOWNERS:22]:
```suggestion
/sdk/internal/           @rickwinter @chlowell @jhendrixMSFT @richardpark-msft @seankane-msft
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 # Catch all for non-code project files and unowned files | folders
 *                        @rickwinter @jhendrixMSFT
-/sdk/                    @rickwinter @chlowell @jhendrixMSFT @richardpark-msft @seankane-msft
+/sdk/                    @rickwinter @chlowell @richardpark-msft @seankane-msft
```
Comment (RickWinter) [inline .github/CODEOWNERS:10]: This is a catch-all for packages that don't have owners.  Are there notifications that you see that you would like to filter out?  Perhaps we can pattern those out.

**Review by RickWinter — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (jhendrixMSFT) [reply .github/CODEOWNERS]: fixed

File: .github/CODEOWNERS (hunk 2 above)
Comment (jhendrixMSFT) [reply .github/CODEOWNERS:10]: Yes I don't want to see any PR notifications from all track 2 SDKs.  It's blowing up my PR folder so I can't track what actually needs my attention.

Comment (RickWinter) [reply .github/CODEOWNERS:10]: Ok, this is fine for now.  I'll work on a solution to minimize what you see without turning everything off :)

Comment (jhendrixMSFT) [reply .github/CODEOWNERS:10]: Filtering out `/sdk/**/arm*` would be good.  Is there a way to describe that?

**Pull request merged**